src/components/PersonCard/PersonCard.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles, Typography } from '@material-ui/core';
+import personIcon from '../../img/popups/NetzentwicklungPopup/person.svg';
+import mailIcon from '../../img/popups/NetzentwicklungPopup/mail.svg';
+import phoneIcon from '../../img/popups/NetzentwicklungPopup/phone.svg';
+
+const useStyles = makeStyles((theme) => {
+  return {
+    row: {
+      minWidth: 250,
+      alignItems: 'center',
+      display: 'flex',
+      '& img': {
+        height: 24,
+        width: 24,
+        marginRight: 5,
+      },
+    },
+    card: {
+      flex: 1,
+      border: '1px solid #ddd',
+      padding: theme.spacing(1),
+      margin: `${theme.spacing(1)}px 0`,
+      borderRadius: 2,
+    },
+  };
+});
+
+const PersonCard = ({
+  title,
+  name,
+  email,
+  phone,
+  division,
+  otherDetails,
+  className,
+}) => {
+  const classes = useStyles();
+  return (
+    <div
+      key={`${title}-${name}`}
+      className={`wkp-person-card ${classes.card}${` ${className}` || ''}`}
+    >
+      {title && <Typography paragraph>{title}</Typography>}
+      {name && (
+        <div className={classes.row}>
+          <img src={personIcon} alt="Person" />
```

Comment:
Cna use use a ReactComponent instead  of an  image ?

Reply:
Yeah, or generally a node or element instead of the image path as prop. That will make it a lot more flexible